mitodl/ol-infrastructure
# config: Add/Enable edx-username-changer plugin feature flag

### Related Ticket:
N/A

### What this PR does?
- Adds/Enables a feature flag for the usage of https://github.com/mitodl/edx-username-changer

### How to test?
- Install the plugin as mentioned https://github.com/mitodl/edx-username-changer#installing-plugin
- Make sure that `ENABLE_EDX_USERNAME_CHANGER` is set to true
- Restart LMS (`make lms-restart`)
- Open LMS/admin and open any user and make sure that the username field is editable successful once changed.
